src/mon/MonCommands.h

```diff
@@ -750,6 +750,14 @@ COMMAND("osd set " \
 COMMAND("osd unset " \
 	"name=key,type=CephChoices,strings=full|pause|noup|nodown|noout|noin|nobackfill|norebalance|norecover|noscrub|nodeep-scrub|notieragent|nosnaptrim", \
 	"unset <key>", "osd", "rw", "cli,rest")
+COMMAND("osd setall " \
+	"type=CephString,name=keys,n=N", \
+	"setall key <key> [<keys>...]", \
+    "osd", "rw", "cli,rest")
+COMMAND("osd unsetall " \
+	"type=CephString,name=keys,n=N", \
+	"unsetall key <key> [<keys>...]", \
+    "osd", "rw", "cli,rest")
```

Comment:
what's the argument for creating a new set of commands for multiple-flag (un)set, instead of just reusing the existing ones and allowing them to take multiple flags?

Reply:
The main question is whether or not we want to allow a single "--yes-i-really-mean-it" to affect all flags in a group. I so far don't see how through the command line parser to associate a flag with a single parameter (e.g. "set foo --yes-i-really-mean-it bar --yes-i-really-mean-it bax baz", where bax and baz are not flagged, OR "set foo bar bax baz --yes-i-really-mean-it", where all flags are affected by the modifier). If the latter is fine, then things are straightforward and I'm completely on board keeping the same command names (and in fact that's practically how it's implemented now). If not, then this is trickier because we'd be changing the semantics, which is the assumption currently made.

Reply:
Sounds like a single YIRM flag was the email consensus and how you're planning to move forward on this PR, right?

Reply:
@gregsfortytwo Yes, and yes!